pages/dkp/kommander/1.4/projects/platform-services/platform-services-catalog/kudo-spark/index.md

```diff
@@ -0,0 +1,512 @@
+---
+layout: layout.pug
+navigationTitle: KUDO Spark
+render: mustache
+title: KUDO Spark
+menuWeight: 3
+excerpt: Day 2 Operations of KUDO Spark
+beta: false
+---
+
+<!-- markdownlint-disable MD018 -->
+## KUDO Spark
+
+Kommander Catalog adds integration for [KUDO Spark Operator](https://github.com/mesosphere/kudo-spark-operator/), which simplifies day 2 operations of [Apache Spark](https://spark.apache.org/). 
+
+#include /dkp/kommander/1.4/include/kudo-intro.tmpl
+
+It is **strongly recommended** to view the [KUDO Spark Documentation](https://github.com/mesosphere/kudo-spark-operator/tree/master/kudo-spark-operator/docs/latest) which covers the KUDO Spark Operator in-depth. This document covers the integration aspects of KUDO Spark Operator with D2iQ Kommander.
```

Comment:
Do we really need to have `D2iQ` here? I feel like this is a given in this docs site. My two cents.

```suggestion
It is **strongly recommended** to view the [KUDO Spark Documentation](https://github.com/mesosphere/kudo-spark-operator/tree/master/kudo-spark-operator/docs/latest) which covers the KUDO Spark Operator in-depth. This document covers the integration aspects of KUDO Spark Operator with Kommander.
```